### ISSUE TITLE
Tests using `web-test-runner` time out waiting for the browser to start

### ISSUE THREAD

**Issue by OBe95** [angular/angular-cli#27447]
### Command

test

### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

We are concerned about Karma not being maintained and we want to know the cost of migrating to other alternatives

We came across your blog post [Moving Angular CLI to Jest and Web Test Runner](https://blog.angular.io/moving-angular-cli-to-jest-and-web-test-runner-ef85ef69ceca) and want to explore the `web-test-runner`, but it fails with the following error: `The browser was unable to create and start a test page after 30000ms. You can increase this timeout with the browserStartTimeout option. `

<img width="1060" alt="image" src="https://github.com/angular/angular-cli/assets/11639464/b2ef6b4a-f0f0-4272-b5ec-b85cbfae1389">

### Minimal Reproduction

I started a new project:
- `ng new test-web-runner`
- selected `SCSS` and `No SSR`
- installed `@web/test-runner`
- switched the `test` builder in `angular.json` to `@angular-devkit/build-angular:web-test-runner`
- run the default test

### Your Environment

```text
Angular CLI: 17.3.3
Node: 20.11.0
Package Manager: npm 10.2.4
OS: darwin arm64

Angular: 17.3.3
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1703.3
@angular-devkit/build-angular   17.3.3
@angular-devkit/core            17.3.3
@angular-devkit/schematics      17.3.3
@schematics/angular             17.3.3
rxjs                            7.8.1
typescript                      5.3.3
zone.js                         0.14.4
```

**Comment by abderrahmane-hilali** [comment]
just install ```@web/test-runner-core``` and it will work.

I think it should be added to the error message

**Comment by berkon** [comment]
Unfortunately installing `@web/test-runner-core` didn't help.

**Comment by OBe95** [comment]
Normally `@web/test-runner-core` is already a [dependency](https://www.npmjs.com/package/@web/test-runner?activeTab=dependencies) of `test-web-runner`

**Comment by jahorton** [comment]
What sort of browser-launcher setup are you using when running your tests?  Are you trying to test against an older set of browsers, by chance?

Apologies if this ends up being unrelated, but I ran into this issue (and error message) while trying to resolve my own; perhaps notes I left at https://github.com/modernweb-dev/web/issues/2756 might help.

**Comment by terrycoughlan** [comment]
Add this to your angular.json file to get the tests passing for now
![image](https://github.com/angular/angular-cli/assets/2589463/ff8163fb-3554-4eb9-84db-1b26c251ccec)

**Comment by berkon** [comment]
Correct, I found this too. Looks like this is required starting with Angular 15.

**Comment by b-ferrari** [comment]
> Add this to your angular.json file to get the tests passing for now ![image](https://github.com/angular/angular-cli/assets/2589463/ff8163fb-3554-4eb9-84db-1b26c251ccec)

Thank you, that solved a similar issue I was having and I'm at least able to get the tests running now!

**Comment by alan-agius4** [comment]
This experimental builder will be removed in version 22.
